src/main/java/io/github/moulberry/notenoughupdates/commands/repo/ResetRepoCommand.java

```diff
@@ -1,18 +0,0 @@
-package io.github.moulberry.notenoughupdates.commands.repo;
-
-import io.github.moulberry.notenoughupdates.NotEnoughUpdates;
-import io.github.moulberry.notenoughupdates.commands.ClientCommandBase;
-import net.minecraft.command.CommandException;
-import net.minecraft.command.ICommandSender;
-
-public class ResetRepoCommand extends ClientCommandBase {
-
-	public ResetRepoCommand() {
-		super("neuresetrepo");
```

Comment:
can we not delete neuresetrepo?

Reply:
It's pretty much useless

Reply:
You probably should change the message then.